spec/forms/schools/add_participants/add_wizard_spec.rb

```diff
@@ -243,28 +238,33 @@
           expect { form.save! }.to change(ECFParticipantValidationData, :count).by 1
         end
 
-        it "sends a participant the added and validated" do
-          form.save!
-          profile = ECFParticipantValidationData.find_by(trn: form.trn).participant_profile
-          expect(ParticipantMailer).not_to have_received(:participant_added)
-          expect(ParticipantMailer).to have_received(:sit_has_added_and_validated_participant)
-                                         .with(participant_profile: profile, school_name: school_cohort.school.name)
+        it "does not send a participant added mail" do
+          expect { form.save! }.not_to have_enqueued_mail(ParticipantMailer, :participant_added)
+        end
+
+        it "sends the SIT an email saying they have been added and validated" do
+          expect { form.save! }
+            .to have_enqueued_mail(ParticipantMailer, :sit_has_added_and_validated_participant)
+              .with(
+                params: {
+                  school_name: school_cohort.school.name,
+                },
+                args: [],
+              )
         end
       end
 
       context "A SIT is adding themselves as a mentor and type is not self" do
         it "does not send the SIT an email saying they have been added and validated" do
           create(:induction_coordinator_profile, user:)
-          form.save!
-          expect(ParticipantMailer).not_to have_received(:sit_has_added_and_validated_participant)
+          expect { form.save! }.not_to have_enqueued_mail(ParticipantMailer, :sit_has_added_and_validated_participant)
         end
       end
 
       context "A SIT is adding themselves as a mentor and type is set to self" do
         it "does not send the SIT an email saying they have been added and validated" do
           form.type = :self
-          form.save!
-          expect(ParticipantMailer).not_to have_received(:sit_has_added_and_validated_participant)
+          expect { form.save! }.not_to have_enqueued_mail(ParticipantMailer, :sit_has_added_and_validated_participant)
```

Comment:
Lovely. Much better using :have_enqueued_mail matcher!

Reply:
Thank @slorek for tidying all these up! 😅 👏🏽